src/CONST.js

```diff
@@ -477,6 +477,7 @@ const CONST = {
     EMOJI_PICKER_ITEM_HEIGHT: 40,
     EMOJI_PICKER_HEADER_HEIGHT: 38,
     COMPOSER_MAX_HEIGHT: 125,
+    CHAT_FOOTER_MIN_HEIGHT: 65,
```

Comment:
Why 65 out of curiosity? We're big fans of numbers divisible by 4 so just wondering :)

Reply:
this was actually taken from [here](https://github.com/Expensify/App/pull/10140/files#diff-8ab8ba45ab927f1430b8d1209d187e003a9cdfe476f809e4483ef64cdba253ebL1370)

Reply:
Ah cool if it already exists, no worries then. Thanks for listening!